gm4_mending_tanks/data/gm4/tags/functions/init_check.json

```diff
@@ -0,0 +1,6 @@
+{
+    "values":[
+        "liquid_tanks:init_check"
+    ]
```

Comment:
Why does this datapack contain function tags that call liquid tanks? These are already part of the liquid tanks base module. Same with `pulse_check.json`.

Reply:
Should be `mending_tanks:...` I'll fix those